### ISSUE TITLE
Merge Hebrew script sources

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans#47]
Merge the Hebrew script design work by @fontef and @danielgrumer into the upstream source files.

TODO:

- [x] update bracket layer definitions for opsz axis changes in #60 (see https://github.com/googlefonts/googlesans/pull/56#issuecomment-647749754 for the definitions that need to be updated)
- [x] merge outlines for min opsz upright masters
- [x] merge outlines for min opsz italics masters
- [x] merge outlines for max opsz upright masters
- [x] merge outlines for max opsz italics masters
- [x] merge fea code for upright source
  - [x] languagesystem (hebr dflt + hebr IWR)
  - [x] global class definitions (renamed with hb_* prefix)
  - [x] aalt
  - [x] ccmp (must be first fea implemented)
  - [x] dlig (Yiddish rules)
  - [x] jalt
  - [x] kern
- [x] merge fea code for italics source
  - [x] languagesystem (hebr dflt + hebr IWR)
  - [x] global class definitions (renamed with hb_* prefix)
  - [x] aalt
  - [x] ccmp (must be first fea implemented)
  - [x] dlig (Yiddish rules)
  - [x] jalt
  - [x] kern
- [x] add Hebrew Unicode range setting in Font > Info tab
- [ ] update expected fea tags with new features that are merged with Hebrew glyph set
- [ ] update fontbakery char set QA tests with new Hebrew glyphs
- [ ] designer review of sources (review in progress)
- [ ] designer review of build artifacts (review in progress)

The review identified the following issues:

- [ ] H spacing is slightly off for some glyphs, more detailed review in progress to understand this problem
- [x] we support tab and caps variants of currency symbols, these new outlines and feature code support will be added to the sources

**Comment by chrissimpkins** [comment]
@danielgrumer beginning the merge this week. I'll be in touch as questions arise.

**Comment by chrissimpkins** [comment]
@danielgrumer Any concerns about renaming your global fea glyph class definitions so that they include a `hb_*` prefix?  I am thinking through how to organize the class definitions and associated scripts as we merge multiple scripts.  This might be a useful indicator of what links where.  This naming approach is valid by the fea spec.  We'll need to replace all of the class reference strings in the rules which I can do on this end.

**Comment by danielgrumer** [comment]
Sounds like a good idea 👍

Should we also do that in the Hebrew repo?

**Comment by chrissimpkins** [comment]
I'll play with it when I work on the fea merge next week and can push the changes downstream to you.  No need to duplicate this.  I'll push updated files based on the LGC sources here for your review once the fea code is in place and before I pull in the outlines.

**Comment by chrissimpkins** [comment]
Opened https://github.com/googlefonts/googlesans/pull/56 for the merge.

Upright fea code merged into source files in https://github.com/googlefonts/googlesans/pull/56/commits/297d855df89de583c2911ac9b1d4f4fbae2ae236 

This included the following definitions and edits of the downstream sources:

-  languagesystem (hebr dflt + hebr IWR)
-  global class definitions renamed with `hb_*` prefix (and updated references in all features)
-  aalt
-  ccmp
-  dlig (Yiddish rules)
-  jalt
-  kern

**Comment by chrissimpkins** [comment]
Italics fea code merged into source files in eeedf9b9e5c6f4be228673ea0fef28571bac2480

This included the following definitions and edits of the downstream sources:

languagesystem (hebr dflt + hebr IWR)
global class definitions renamed with `hb_*` prefix (and updated references in all features)
aalt
ccmp
dlig (Yiddish rules)
jalt
kern

**Comment by chrissimpkins** [comment]
Currently blocked on #49 work.  Will resume work here when the opsz axis changes are defined.

**Comment by chrissimpkins** [comment]
Upright and italic outlines, kerning, and feature code are merged as of https://github.com/googlefonts/googlesans/pull/56/commits/3234868c49fd18338679306acafda687d9914cfe in PR #56

**Comment by chrissimpkins** [comment]
Hebrew Unicode range added in https://github.com/googlefonts/googlesans/commit/b940cc563ae3cf02e656445c90ca09e3b396156c
